src/Chip/Chip.tsx

```diff
@@ -90,7 +90,8 @@ const Container = styled(Box)<IButton>(
     justify-content: ${$icon ? 'space-evenly' : 'center'};
     line-height: 100%;
     padding: 8px 16px 8px ${$icon ? '8px' : '16px'};
-    width: 98px;
+    min-width: 98px;
+    max-width: 120px;
```

Comment:
Is there a reason why we give it a `max-width`?

Does this not make this component not very flexible when used for chips with long text?

Reply:
Another point, I'm guessing `max-width` would be problematic for users that use big text (for accessibility)

Reply:
good point tbf, by design these chips are supposed to be small with one word text such as "add"/"added" but and we shouldn't be using it for anything larger. That being said, I've removed the constraint incase we find ourselves in that position 😅 

![Screenshot 2024-10-16 at 10 11 38](https://github.com/user-attachments/assets/0a1fa21a-e940-4446-a151-c97e198d1d8f)